src/system/system.kathleen.ucl.ac.uk.make

```diff
@@ -13,7 +13,7 @@ OMPFLAGS= -fopenmp
 
 # Compilation flags
 # NB for gcc10 you need to add -fallow-argument-mismatch
-COMPFLAGS= -O3 -g $(OMPFLAGS) $(XC_COMPFLAGS) -I"${MKLROOT}/include"
+COMPFLAGS= -xAVX -O3 -g $(OMPFLAGS) $(XC_COMPFLAGS) -I"${MKLROOT}/include"
```

Comment:
Could you move the modifications to `system.kathleen.*.make` into a separate PR that closes #308 ?

It doesn't make much difference here but in general using `-xHost` is better than `-xAVX`. `-xHost` will select the highest level of vectorization supported by the hardware. Yes I know I told you to use `-xAVX`, it's my fault :see_no_evil: